furusiya/model/entities/party/player.py

```diff
@@ -48,14 +48,26 @@ def unmount(self, horse):
             self.mounted = False
             horse.is_mounted = False
 
+    def _rest(self, current_hp, max_hp):
+        return int(config.data.player.restingPercent * max_hp / current_hp)
```

Comment:
This formula still doesn't make sense. Assuming `restingPercent` is 10%, `max_hp` is `20`, here's what I get for different values of `current_hp`:

- 20: `2/20` or `0.1`
- 15: `2/15` or `0.1333`
- 10: `2/10` or `0.5`
- 3: `2/3` or `0.667`
- 1: `2/1` or `2`

Please remove the part where you divide by `current_hp`